webapp/apps/core/views.py

```diff
@@ -0,0 +1,178 @@
+from django.shortcuts import render, get_object_or_404
+from django.http import JsonResponse
+
+from django.utils import timezone
+from .models import CoreRun
+from ..taxbrain.models import OutputUrl
+from .compute import Compute, JobFailError
+from ..formatters import get_version
+from ..taxbrain.views import TAXCALC_VERSION, WEBAPP_VERSION, dropq_compute
+from django.shortcuts import (render, render_to_response, get_object_or_404,
+                              redirect)
+from django.template.context import RequestContext
+
+
+def output_detail(request, pk):
+    """
+    This view is the single page of diplaying a progress bar for how
+    close the job is to finishing, and then it will also display the
+    job results if the job is done. Finally, it will render a 'job failed'
+    page if the job has failed.
+
+    Cases:
+        case 1: result is ready and successful
+
+        case 2: model run failed
+
+        case 3: query results
+          case 3a: all jobs have completed
+          case 3b: not all jobs have completed
+    """
+
+    try:
+        url = OutputUrl.objects.get(pk=pk)
+    except OutputUrl.DoesNotExist:
+        raise Http404
+
```

Comment:
@lucassz what do you think about doing the URL routing through the new outputs table using the `uuid` variable?